std_msgs/Header header

float32 roll	# roll angle [rad]
float32 pitch	# pitch angle [rad]
float32 yaw		# yaw angle [rad]
float32 gaz 	# vertical velocity [m/s]
